modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowServiceImpl.java

```diff
@@ -2278,7 +2278,7 @@ private void removeWorkflowInstanceFromIndex(long workflowInstanceId, Elasticsea
    * @param index
    *         the index to update
    */
-  private void updateWorkflowInstanceInIndex(long id, int state, String mpId, String orgId, ElasticsearchIndex index) {
+  private void updateWorkflowInstanceInIndex(long id, int state, String mpId, String orgId) {
```

Comment:
This isn't really consistent with the other methods anymore (see `remove...`). I'd just remove the index from all of them.